pkg/apis/componentconfig/edgecore/v1alpha1/types.go

```diff
@@ -388,6 +388,13 @@ type MetaManager struct {
 	// RemoteQueryTimeout indicates remote query timeout (second)
 	// default 60
 	RemoteQueryTimeout int32 `json:"remoteQueryTimeout,omitempty"`
+	// The config of MetaServer
+	MetaServer `json:"metaServer,omitempty"`
+}
+
+type MetaServer struct {
+	Enable bool
+	Debug  bool
```

Comment:
We'd better add the `json:"metaServer,omitempty"` here

Reply:
And do we have set the default value for the MetaServer here?

Reply:
@fisherxu looks good, I will add.